src/other/PropertyEditor.js

```diff
@@ -74,6 +81,21 @@
         ;
         table
             .each(function (d) {
+                if (d instanceof PropertyEditor === false) {
+                    if (context.overrideTags() && Object.keys(context.overrideTags()).length > 0) {
+                        var proto = Object.getPrototypeOf(d);
+                        for (var key in context.overrideTags()) {
+                            if (d._class.indexOf(key) !== -1) {
+                                for (var param in context.overrideTags()[key]) {
+                                    if (proto["__meta_" + param].ext && proto["__meta_" + param].ext.tags) {
+                                        proto["__meta_" + param].ext.tags = context.overrideTags()[key][param];
```

Comment:
I suspect you could add a method in PropertyExt to facilitate this (I am still not sure what "this" is...)